src/extensions/renderer/canvas/drawing-nodes.js

```diff
@@ -264,24 +265,48 @@ CRp.drawNode = function( context, node, shiftToOriginWithBb, drawLabel = true, s
 
     context.translate( gx, gy );
 
-    setupShapeColor( ghostOpacity * bgOpacity );
+    if ( bgContainment === 'inside' ){
```

Comment:
This containment property looks great.  However, I could see some conflicts with certain use cases.  The main issue is that people would like to set this on a per-image basis, e.g. one image as the true background and one as an overlay.

This property could be per-image, like most other background image properties.  The `drawImages()` function could be parameterised to filter on only the specified subset of images.  That would keep things simple.

Reply:
N.b. if only one value is specified, then that value should be applied to all images.

E.g. [inside, over, inside] v.s. inside

Reply:
We are changing the complete drawing order based on the `background-image-containment` property:

Inside:

1. Shape Color
2. Shape
3. Image
4. Pie (deprecated)
5. Border Color
6. Border
7. Darken

Over:
1. Shape Color
2. Shape
3. Border Color
4. Border
5. Pie (deprecated)
6. Images
7. Darken

I can not see how we can introduce the multiple enum value to change the drawing order for some styles only.